src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -143,6 +144,21 @@ void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
   LOG4CXX_DEBUG(
       logger_, "Header added: " << std::string(message.begin(), message.end()));
 }
+
+std::string OnSystemRequestNotification::ParsePTString(std::string& pt_string) const{
+  std::string result;
```

Comment:
@AGaliuzov This will be more efficient then using std::string insert() every time those characters are detected. But if you would rather this modify the the same string instead of returning a copy I can change ParsePTString to
```
void OnSystemRequestNotification::ParsePTString(std::string& pt_string) const{
  std::string result;
  size_t length = pt_string.length();  
  result.reserve(length*2);
  for(size_t i=0;i<length;++i){
    if(pt_string[i]=='\"' || pt_string[i]=='\\'){
      result += '\\';
    } else if(pt_string[i] == '\n') {
      continue;
    }
    result += pt_string[i];
  }
  pt_string.reserve(result.length());
  pt_string = result;
}
```